docs/build/apps/stellar-wallets-kit.mdx

```diff
@@ -0,0 +1,136 @@
+
+## Compattible Wallets:
+
+The following wallets are compatible with the Stellar Wallets Kit:
+
+- xBull Wallet (both PWA and extension versions)
+- Albedo
+- Freighter
+- Rabet (extension version)
+- WalletConnect
+- Lobstr
+- Hana
+
+## Installation
+
+To install the library, use npm:
+
+<CodeExample>
+  ```bash npm install @creit.tech/stellar-wallets-kit ```
+</CodeExample>
+
+## Usage
+
+### Creating an Instance
+
+1. First, create an instance of the `StellarWalletsKit` class. You should only create one instance to avoid unexpected behavior. Here’s an example:
+
+<CodeExample >
+```javascript
+import {
+  StellarWalletsKit,
+  WalletNetwork,
+  allowAllModules,
+  XBULL_ID,
+} from '@creit.tech/stellar-wallets-kit';
+
+const kit = new StellarWalletsKit({ network: WalletNetwork.TESTNET, selectedWalletId: XBULL_ID, modules: allowAllModules(), });
+
+````
+</CodeExample>
+The `allowAllModules()` function imports modules where no additional configuration is needed (e.g., WalletConnect). If you want to specify only certain wallets to support, pass an array of modules to the constructor like this :
+<CodeExample >
+```javascript
+import {
+  FreighterModule,
+  StellarWalletsKit,
+  WalletNetwork,
+  XBULL_ID,
+  xBullModule
+} from '@creit.tech/stellar-wallets-kit';
+
+const kit: StellarWalletsKit = new StellarWalletsKit({
+  network: WalletNetwork.TESTNET,
+  selectedWalletId: XBULL_ID,
+  modules: [
+    new xBullModule(),
+    new FreighterModule(),
+  ]
+});
+````
+
+</CodeExample>
+
+## Integrated UI Modal
+
+The library includes a UI modal that you can show your users after initializing the kit. Once they select a wallet, you can use other available methods. Here’s how to use it:
+
+<CodeExample >
+```javascript
+await kit.openModal({
+  onWalletSelected: async (option) => {
+    kit.setWallet(option.id);
+    const publicKey = await kit.getPublicKey();
+    // Perform additional actions
+  },
+});
+```
+</CodeExample>
+The `openModal` method allows you to customize the modal title, allowed wallets, and styles.
+<CodeExample>
+```javascript
+function openModal(params: {
+    onWalletSelected: (option: ISupportedWallet) => void;
+    onClosed?: (err: Error) => void;
+    modalDialogStyles?: { [name: string]: string | number | undefined | null; }
+    modalTitle?: string;
+    notAvailableText?: string;
+}) {}
+```
+</CodeExample>
+
+## How to request the public key and sign transactions
+
+Each wallet has its own way when it comes to both requesting the public key and signing a transaction. The kit allows you perform both actions with a unified API:
```

Comment:
In theory, all the wallets should now conform to the wallet standard interface in the future. I'd point out this here https://github.com/stellar/stellar-protocol/discussions/1467

Reply:
Hi @janewang , i am confused about the changes i am to make here . Could you point me to then right direction ?

Reply:
@od-hunter There isn't much to do right now because the stellar-wallet-kit needs to update to the wallet standard first. I just posted here as a FYI